code/src/containers/products-container/ProductsContainer.scss

```diff
@@ -2,9 +2,21 @@
 
 .products-container {
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+  //grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
```

Comment:
remove the comment here

Reply:
Thanks for noticing. I entirely removed changes from that file because it is not related to my pr, I just tested how to make product grid better there